raiden-ts/tests/unit/raiden.spec.ts

```diff
@@ -840,4 +899,70 @@ describe('Raiden', () => {
     const raiden = new Raiden(dummyState, deps, combineRaidenEpics([initEpicMock]), dummyReducer);
     await expect(raiden.getBlockNumber()).resolves.toEqual(blockNumber);
   });
+
+  test('depositChannel', async () => {
+    function channelDepositEpicMock(action$: Observable<RaidenAction>) {
+      return action$.pipe(
+        filter(channelDeposit.request.is),
+        mergeMap(() => {
+          return of(
+            channelDeposit.success(
+              {
+                id: channelId,
+                participant: address as Address,
+                totalDeposit: deposit as UInt<32>,
+                txHash,
+                txBlock: 11,
+                confirmed: true,
+              },
+              { tokenNetwork, partner },
+            ),
+            messageServiceSend.request(
```

Comment:
Can't this mocked action be deduplicated from the `openChannel` test?

Reply:
Done I have taken these actions out so that they could be shared between `openChannel ` and `depositChannel`